docs/models.rst

```diff
@@ -226,3 +226,5 @@ The convergence map is used to lens a Gaussian realization of the unlensed prima
 The primary CMB is obtained from parameters that match the Websky simulation, namely A_s = 2.022e-9, tau = 0.055, and all other parameters set to the websky values above, see `the code <https://github.com/ajvanengelen/webskylensing/blob/master/py/get_cmb_powerspectra.py>`_, values not explicitely set are equal to the CAMB default, for example :math:`N_{eff}=3.046` .  The CAR maps (where the lens remapping is done) have 1 arcminute resolution.  
 
 The theoretical power spectra for the unlensed and lensed CMB are available here https://github.com/ajvanengelen/webskylensing/tree/master/data.  Each is a numpy array of shape `(3, 3, N_l)`, giving the theory power spectrum C_l's in the order ((TT, TE, TB), (ET, EE, EB), (BT, BE, BB)) in units of uK_CMB^2.  They are obtained from the `get_cmb_powerspecta.websky_cmb_spectra` routine in that repository, which serves as a wrapper to CAMB.
+
+The :py:class:`WebSkyCMBTensor` provides the $BB$ spectrum for the Websky cosmology from a model with $r = 1$ (which of course needs to be scaled to whatever actual $r$ value we want to use). This component is not lensed and we consider only $C_\ell^{BB}$.
```

Comment:
@ajvanengelen can you please suggest any improvements to the docs?

Reply:
Hi, looks good, maybe add something saying that temperature fluctuations from tensors are not included?

Reply:
ok